### PR TITLE
Add logistics for ONNX 1.13 release

### PR DESCRIPTION
**Pull request by p-wysocki**
Signed-off-by: p-wysocki <przemyslaw.wysocki@intel.com>

### Description
- Currently filled with PRs merged between ONNX 1.12 release and 17th November
- Will be updated in real time until ONNX 1.13 release 

[ONNX Wiki](https://github.com/onnx/onnx/wiki) states that it will be moved into `onnx/docs`. Should the release files be loose in the general directory or should I create a subdirectory for release files?

### CODE REVIEW

File: docs/Logistics-for-ONNX-Release-1.13.0.md
Comment (p-wysocki) [reply docs/Logistics-for-ONNX-Release-1.13.0.md]: Good point, I removed the deprecation message.

**Review by xadupre — APPROVED** [review]

**Review by jcwchen — COMMENTED** [review]
Why don't we still use wiki to handle logistics: https://github.com/onnx/onnx/wiki? I thought this PR is only for review purpose.

**Comment by p-wysocki** [comment]
@jcwchen I created this document as the wiki opens with the following message
image
Should use the wiki anyway?

**Review by jcwchen — COMMENTED** [review]
Thanks for the info -- I didn't notice that announcement, but it's quite old (announced in 2019 and there are still logistics for 1.9 to 1.12.0 releases in wiki). I am fine with either way. The advantage of wiki would be it's much easier for the release manager to modify the document content.

**Comment by p-wysocki** [comment]
@gramalingam do you have an opinion on whether this document should be on wiki or here, in docs directory? I don't mind either.

**Comment by gramalingam** [comment]
I don't mind either way. One advantage to having it in `docs` is to have documents in one place. But @jcwchen why is it easier to update the wiki, is it because it has different approval-requirements?

**Comment by jcwchen** [comment]
> One advantage to having it in docs is to have documents in one place. But @jcwchen why is it easier to update the wiki, is it because it has different approval-requirements?

Yes... I just feel the document will be updated very frequently during release period, but yes we can also keep this PR open until the release is finalized. Having the wiki interface is also more convenient for the editors to update content. Still, I don't have strong preference. @p-wysocki feel free to forward this PR if you think it's better to you. Thanks!

**Comment by p-wysocki** [comment]
@gramalingam @jcwchen I moved the logistics to the wiki, at https://github.com/onnx/onnx/wiki/Logistics-for-ONNX-Release-1.13.0, to keep consistency with previous releases' logistics. If the wiki is in fact going to be moved into `docs/` it can be done in one go, instead of having the documents scattered across two different places.

**Pull request closed without merging**
